strings/text_wrap.py

```diff
@@ -0,0 +1,18 @@
+# Text Wrap
+"""Task
+You are given a string S and width w.
+Your task is to wrap the string into a paragraph of width w.
+"""
+
+
+import textwrap
+
+
+def wrap(string, max_width):
+    return textwrap.TextWrapper(width=max_width).fill(text=string)
```

Comment:
can you implement it without using `textwrap`?

Reply:
Yes:
1.
```
#solution 2
def wrap(string, max_width):
    l = []
    for i in range(0, len(string), max_width):
        l.append(string[i:i + max_width])
    return '\n'.join(l)


if __name__ == '__main__':
    string, max_width = input(), int(input())
    result = wrap(string, max_width)
    print(result)
```
2.
```
# solution 3
def wrap(string, max_width):
    return '\n'.join([string[i:i + max_width] for i in range(0, len(string), max_width)])


if __name__ == '__main__':
    string, max_width = input(), int(input())
    result = wrap(string, max_width)
    print(result)
```

Reply:
can you please replace it in the code, not just in the comment?

Reply:
ok